packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1407,8 +1407,8 @@ export class ComboBox extends React.Component<IComboBoxProps, IComboBoxState> {
           onMouseLeave={this._onOptionMouseLeave}
           role="option"
           // aria-selected should only be applied to checked items, not hovered items
-          aria-selected={isSelected ? 'true' : 'false'}
-          ariaLabel={item.ariaLabel}
+          aria-selected={isChecked ? 'true' : 'false'}
+          ariaLabel={this._getPreviewText(item)}
```

Comment:
In this one case, I'd suggest leaving this as `ariaLabel={item.ariaLabel}`, since defining it as `item._getPreviewText` means the aria-label will always be set, even if an author hasn't defined `item.ariaLabel`. That actually caused an a11y bug in v8 (fixed now) when a team had different option text in the dropdown than the text used in the input value.